controller-server/src/main/java/com/yahoo/vespa/hosted/controller/restapi/application/ApplicationApiHandler.java

```diff
@@ -164,6 +165,7 @@ private HttpResponse handleGET(HttpRequest request) {
         if (path.matches("/application/v4/property")) return properties();
         if (path.matches("/application/v4/cookiefreshness")) return cookieFreshness(request);
         if (path.matches("/application/v4/tenant/{tenant}")) return tenant(path.get("tenant"), request);
+        if (path.matches("/application/v4/tenant/{tenant}/property")) return property(path.get("tenant"));
```

Comment:
As discussed offline: I suggest adding this to the tenant response and removing this path.

Also, what is the reason for exposing this information in the API?

Reply:
It was requested by the dashboard team.

Reply:
I changed my mind on this; 

The extra stuff displayed as propertyInfo requires lookups in OpsDB and JIRA, and I don't want to screw up all the 6 different tenant-renderer-paths when one of these external services fail to reply, time out, or whatever, so I think it's better to put it at a separate location.

Reply:
Perhaps this info should be stored in Tenant, and refreshed periodically by maintainers, as we do with other such information, but I think that's work for later. 

Tenant needs some work, anyway.

Reply:
I'll put the stuff on the tenant path anyway, for now, and make sure loading times are low later, if needed.